### DISCUSSION TITLE
Draggable function is not getting called

### DISCUSSION THREAD

**Discussion by solulabahmad** [GrapesJS/grapesjs#5281] [Q&A]
Hey @artf  and rest of gjs team, I am creating one component and passing function as draggable value but that function is not getting called while component is being dragged. Attaching my code snippet for the reference

  editor.DomComponents.addType('normalVideo', {
        model: {
          defaults: {
            tagName: 'div',
            draggable: () => {
              console.log('Test')
              return true
            }
           }
         }
        }
)

**Comment by artf** [comment]
I don't see any issue with `draggable` property, please provide a reproducible demo.
